### PR TITLE
Add anchors and minor repo enhancements

### PR DESCRIPTION
**Pull request by NickNtamp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,14 +57,15 @@ To set up on your local machine:
...
+| Local Interpretable Model-agnostic (LIME) | model-agnostic | LIME  is a visualization technique that helps explain individual predictions. It is model agnostic so it can be applied to any supervised regression or classification model. | [Quick start](examples/00_quick_start/lime.ipynb) | Google Colab |
+| Layer-wise Relevance Propagation (LRP) | non-model-agnostic | LRP is a technique that brings such explainability and scales to potentially highly complex deep neural networks. It operates by propagating the prediction backward in the neural network, using a set of purposely designed propagation rules | [Quick start](examples/00_quick_start/lrp.ipynbb) | Google Colab |
```
Comment (momegas) [inline README.md:67]: Why is this in Colab?

Comment (NickNtamp) [reply README.md:67]: Thanks @momegas . You have highlighted all the table so I can't understand for what specific model you are talking about. Also nothing is in Colab, Colab is the proposed environment for an external user in order to reproduce the code.

Comment (momegas) [reply README.md:67]: The environment column was added now. This is what I'm referring to. The proposed environment for XAI is local. The code should be run on anyones laptop. 
I think that Colab shouldn't be there unless there is no other choice.

Comment (NickNtamp) [reply README.md]: @momegas please check my last commit

Comment (momegas) [reply README.md]: Looks great Nick. We need to clean some more things here and there but this looks really nice 👍

**Review by momegas — APPROVED** [review]

**Pull request merged**